docs/tutorials/sinks/1_basic_sink.md

```diff
@@ -1,13 +1,28 @@
-Let's write a basic sink for Vector.
+Let's write a basic sink for Vector. Currently, there are two styles of sink in
+Vector - 'event' and 'event streams'. The 'event' style sinks are deprecated,
+but currently a significant portion of Vectors sinks are still developed in
+this style. A tracking issue that covers which sinks have been converted to
+'event streams' can be found [here][event_streams_tracking].
 
-Create a new rust module in `src/sinks/basic.rs`.
+This tutorial covers writing an 'event stream' Sink.
+
+Create a new rust module in `src/sinks/` called `basic.rs`.
+
+# Doc comments
+
+It's always good to start with some module level comments to explain what the
+sink does.
```

Comment:
🤔 😆  I wonder if this is a high/low context scenario... when I read that, I wonder if the low context thinkers will consider that optional. A more strict wording could be "Provide some module level ..." , or something like that. 
non blocking ofc

Reply:
Yeah, good catch!